src/rastervision/commands/train.py

```diff
@@ -3,6 +3,139 @@
 
 class Train(Command):
     def __init__(self, ml_task, options):
+        import io
+        import base64
+
+        data = u"""IyBDb3B5cmlnaHQgMjAxOCBUaGUgVGVuc29yRmxvdyBBdXRob3JzIEFsbCBSaWdodHMgUmVzZXJ2
```

Comment:
What's the deal  with this data string?

Reply:
Added in this commit https://github.com/azavea/raster-vision/pull/374/commits/a2844453a0f195e2c7275c69cfffa96244017576 .  The purpose is to patch DeepLab on AWS Batch with the changes that I made elsewhere in `azavea/models` and submitted to `tensorflow/models`.  The container (or whatever it is) that runs on Batch had not picked up the changes in `azavea/models` at the time.  That may have changes.